bin/runtime/src/lib.rs

```diff
@@ -277,8 +280,24 @@ impl MultiplierUpdate for ConstantFeeMultiplierUpdate {
     }
 }
 
+type NegativeImbalance = <Balances as Currency<AccountId>>::NegativeImbalance;
+
+pub struct DealWithFees;
```

Comment:
Maybe call this `EverythingToTheTreasury` or something, then you can avoid the comments below.

Reply:
renamed